syntax = "proto3";

package rpc;

option cc_generic_services = true;

message ResultCode {
    int32 errcode = 1;
    bytes errmsg = 2;
}

message FriendRequest {
    int32 userid = 1;
}

message FriendResponse {
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendService {
    rpc getFriendList(FriendRequest) returns(FriendResponse);
}

